packages/http/src/validator/validators/body.ts

```diff
@@ -87,7 +87,14 @@ export function parseMultipartFormDataParams(
 
 export function decodeUriEntities(target: Dictionary<string>) {
```

Comment:
no content type available when this is called?

Reply:
nope.